blocks/library/cover-image/index.js

```diff
@@ -131,7 +131,7 @@ registerBlockType( 'core/cover-image', {
 					</MediaUploadButton>
 				</Toolbar>
 			</BlockControls>,
-			<InspectorControls key="inspector">
+			!!url && <InspectorControls key="inspector">
```

Comment:
I think the build error is related to the code styling here. There should be a space after the `!!`